### PR TITLE
Update the min version of curl required by PHP

### PR DESCRIPTION
**Pull request by stof**
PHP 7.3 increased the min version of curl

### CODE REVIEW

File: reference/curl/setup.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <para>
    In order to use PHP's cURL functions you need to install the <link
    xlink:href="&url.curl;">libcurl</link> package. PHP requires libcurl
-   version 7.10.5 or later.
+   version 7.15.5 or later.
```
Comment (cmb69) [inline reference/curl/setup.xml:13]: The manual is currently supposed to document PHP 7.0 - 8.1, so we need to keep the information for the old versions. Also, as of PHP 8.0.0, curl ≥ 7.29.0 is required so maybe:
```suggestion
   version 7.10.5 or later.
   As of PHP 7.3.0, version 7.15.5 or later is required.
   As of PHP 8.0.0, version 7.29.0 or later is required.
```

**Review by cmb69 — CHANGES_REQUESTED** [review]
Thank you for the PR!

**Comment by stof** [comment]
@cmb69 updated

**Comment by cmb69** [comment]
Thank you!

**Pull request merged**